python/src/GaussLegendre_doc.i.in

```diff
@@ -9,34 +9,136 @@ Available constructors:
 Parameters
 ----------
 dimension : int, :math:`dimension>0`
-    The dimension of the functions to integrate. The default discretization is *GaussLegendre-DefaultMarginalIntegrationPointsNumber* in each dimension, see :class:`~openturns.ResourceMap`.
+    The input dimension of the functions to integrate.
 discretization : sequence of int
-    The number of nodes in each dimension. The sequence must be non-empty and must contain only positive values.
+    The number of nodes in each dimension.
+    The sequence must be non-empty and must contain only positive values.
+    The default discretization is *GaussLegendre-DefaultMarginalIntegrationPointsNumber*
+    in each dimension, see :class:`~openturns.ResourceMap`.
 
 Notes
 -----
-The Gauss-Legendre algorithm enables to approximate the definite integral:
+The Gauss-Legendre algorithm approximates the definite integral:
 
 .. math::
 
-    \int_{\vect{a}}^\vect{b} f(\vect{t})\di{\vect{t}}
+    \int_{\vect{a}}^\vect{b} \model(\vect{x})\di{\vect{x}}
 
 
-with :math:`f: \Rset^d \mapsto \Rset^p`, :math:`\vect{a}, \vect{b}\in\Rset^d` using a fixed tensorized Gauss-Legendre approximation:
+where :math:`\model: \Rset^\inputDim \mapsto \Rset^\outputDim` is a function and
```

Comment:
The changes to this file are in direct conflict with the changes proposed in PR https://github.com/openturns/openturns/pull/2766, you probably need to coordinate with @adutfoy